### DISCUSSION TITLE
was wheatley ever in bee 2

### DISCUSSION THREAD

**Discussion by Darealxbox** [BEEmod/BEE2-items#4235] [Tech Support / Q&A]
was wheatley as a boss ever in bee 2 if so what version

**Comment by TeamSpen210** [comment]
What do you mean by "boss"? He's available as a set of voice lines, but if you mean an item to put the whole boss fight into your level, that would be exceptionally pointless.

**Reply by Darealxbox** [reply]
I remembered wheatley being a item in an old version lol I might be getting the boss stuff from animal king
